src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+
+{{site.data.alerts.callout_success}}
+For a practical guide on how CockroachDB uses Raft to replicate, distribute, and rebalance data, refer to the [Replication and Rebalancing]({% link {{ page.version.version }}/demo-replication-and-rebalancing.md %}) demo.
+{{site.data.alerts.end}}
+
+## High availability
+
+- [**Raft replication**]({% link {{ page.version.version }}/architecture/replication-layer.md %}): CockroachDB's built-in Raft replication stores data safely and consistently on multiple nodes to ensure no downtime even during a temporary node outage. [Replication controls]({% link {{ page.version.version }}/configure-replication-zones.md %}) allow you to configure the number and location of replicas to suit a deployment.
+- [**Fault tolerance**]({% link {{ page.version.version }}/demo-fault-tolerance-and-recovery.md %}): Capabilities built in to CockroachDB to perform routine maintenance operations without any affect on performance. For example, [online schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), [WAL failover]({% link {{ page.version.version }}/cockroach-start.md %}#write-ahead-log-wal-failover).
+- **Logical data replication (LDR)** (Preview): A cross-cluster replication tool between multiple active CockroachDB clusters, which supports a diverse range of topologies. LDR provides eventually consistent, table-level replication.
+
+### Choose a high availability strategy
+
+CockroachDB uses synchronous, built-in Raft replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption, whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication (synchronous)</th>
+    <th>Multi-region replication (synchronous)</th>
+    <th>Logical data replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</td>
+    <td>Immediate mode 0.5 seconds<br>Transactional mode ~10s of seconds</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Add application cutover time</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+    <td>Region-local latency depending on design<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+  </tr>
```

Comment:
On what benchmark are the numbers in this table based? It would be useful to include a link.

Reply:
@alicia-l2 / @awoods187 do we have benchmarks we can refer to here?

Reply:
This is just the network latency (speed of light)

Reply:
@nhamlani101 does KV have some benchmarks anywhere? specifically looking for single region write latency, cross-region write latency

Reply:
I'm going to follow-up with this in another PR, so we can get a first version of this page out.